### PR TITLE
Workaround for angular/angular#6501

### PR DESCRIPTION
**Pull request by SethDavenport**
Pull in missing IE shims.

Also added a workaround for the lack of the input placeholder attribute on IE9.

Connected to rangle/rangle-starter#62

### CODE REVIEW

**Comment by SethDavenport** [comment]
FYI @tsungting @concertcoder

File: webpack.config.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ const plugins = basePlugins
 module.exports = {
   entry: {
     app: './src/index.ts',
+    shims: './shims/shims_for_IE',
```
Comment (nexus-uw) [inline webpack.config.js]: should the other shims be pulled into this shim.js file or left in the vendor file?

**Comment by nexus-uw** [comment]
it is working for me on IE11. :shipit:

File: webpack.config.js (hunk 1 above)
Comment (SethDavenport) [inline webpack.config.js]: yeah we could potentially do that - those shims are general though and these ones are IE specific.  I'd have to make sure I don't break the Karma setup.

**Pull request merged**